source/gameengine/Rasterizer/RAS_CubeMap.cpp

```diff
@@ -66,10 +60,16 @@ static const MT_Matrix4x4 backFaceViewMat(
 	1.0f, 0.0f, 0.0f, 0.0f,
 	0.0f, 0.0f, 0.0f, 1.0f);
 
-static const MT_Matrix4x4 frontFaceViewMat(
-	0.0f, 0.0f, -1.0f, 0.0f,
+static const MT_Matrix4x4 rightFaceViewMat(
+	1.0f, 0.0f, 0.0f, 0.0f,
+	0.0f, 0.0f, 1.0f, 0.0f,
 	0.0f, -1.0f, 0.0f, 0.0f,
-	-1.0f, 0.0f, 0.0f, 0.0f,
+	0.0f, 0.0f, 0.0f, 1.0f);
+
+static const MT_Matrix4x4 leftFaceViewMat(
+	1.0f, 0.0f, 0.0f, 0.0f,
+	0.0f, 0.0f, -1.0f, 0.0f,
+	0.0f, 1.0f, 0.0f, 0.0f,
 	0.0f, 0.0f, 0.0f, 1.0f);
 
 
```

Comment:
The commit message said that render matrix are fixed but not culling matrix. You should not have any issue with world background rendering but culling maybe be wrong.